app/main.py

```diff
@@ -1,6 +1,17 @@
 from typing import Callable
+from typing import Any
```

Comment:
There is no need to duplicate `from typing import`, import `Any` together with `Callable` on the first line.